frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -13,18 +13,19 @@ import {
 import Button from '../Common/Button/Button';
 import Input from '../Common/Input/Input';
 import Switch from '../Common/Switch/Switch';
+import Selector from '../Common/Selector/Selector'; // adjust path as needed
```

Comment:
```suggestion
import Selector from '../Common/Selector/Selector';
```

not sure we need that comment?